components/calendar/src/cal/hijri.rs

```diff
@@ -901,12 +901,6 @@ impl<R: Rules> Calendar for Hijri<R> {
             year = self.0.year_data(year.extended_year + 1)
         }
 
-        // Clamp the RD to our year
```

Comment:
question: why is this no longer needed? Only because now we're 100% sure we're in well-behaved range?